skt/executable.py

```diff
@@ -443,11 +445,19 @@ def cmd_run(cfg):
     run_report_path = join_with_slash(cfg.get('output_dir'), 'run.report')
 
     runner = skt.runner.getrunner(*cfg.get('runner'))
-    retcode, report_string = runner.run(cfg.get('buildurl'),
-                                        cfg.get('max_aborted_count'),
-                                        cfg.get('krelease'),
-                                        cfg.get('wait'),
-                                        arch=cfg.get("kernel_arch"))
+
+    atexit.register(runner.cleanup_handler)
+    signal.signal(signal.SIGINT, runner.signal_handler)
+    try:
```

Comment:
All of the `runner.run` code that's dealing with Beaker (from job submission to waiting for completion) is already wrapped in a `try/except` block: https://github.com/CKI-project/skt/blob/master/skt/runner.py#L468

In theory, it is possible that the assignments and result interpretation that aren't in the block would cause an exception, but we don't want to catch them because the issues would be interpreted as a completed runs. Can you elaborate what advantage the addition of the block adds here?

Reply:
OK, probably not the best way to handle this part. There's quite a bit of code inside that isn't wrapped in any exception, but we can address that later on.